src/Model_Hydro/CPU_Hydro/CPU_Shared_DataReconstruction.cpp

```diff
@@ -1993,27 +2002,33 @@ void Hydro_HancockPredict( real fc[][NCOMP_LR], const real dt, const real dh,
       if ( fc[f][4] <= (real)0.0 || fc[f][4] >= HUGE_NUMBER || fc[f][4] != fc[f][4] ) reset_cell = true;
       if ( Pres     <= (real)0.0 || Pres     >= HUGE_NUMBER || Pres     != Pres     ) reset_cell = true;
 #     endif // #ifndef BAROTROPIC_EOS
+#     endif // #ifdef SRHD
 
 //    set to the cell-centered values before update
       if ( reset_cell )
       {
          for (int face=0; face<6; face++)
          for (int v=0; v<NCOMP_TOTAL; v++)
-            fc[face][v] = g_cc_array[v][cc_idx];
+            fcCon[face][v] = g_cc_array[v][cc_idx];
 
          break;  // no need to apply the floors since the input values should already satisfy these constraints
       }
 
+#ifndef SRHD
+   for (int f=0; f<6; f++)
+   {
 //    apply density and internal energy floors
-      fc[f][0] = FMAX( fc[f][0], MinDens );
+      fcCon[f][0] = FMAX( fcCon[f][0], MinDens );
 #     ifndef BAROTROPIC_EOS
-      fc[f][4] = Hydro_CheckMinEintInEngy( fc[f][0], fc[f][1], fc[f][2], fc[f][3], fc[f][4], MinEint, Emag );
+      fcCon[f][4] = Hydro_CheckMinEintInEngy( fcCon[f][0], fcCon[f][1], fcCon[f][2], fcCon[f][3], fcCon[f][4], 
+                                              MinEint, Emag );
 #     endif
 #     if ( NCOMP_PASSIVE > 0 )
-      for (int v=NCOMP_FLUID; v<NCOMP_TOTAL; v++) fc[f][v] = FMAX( fc[f][v], TINY_NUMBER );
+      for (int v=NCOMP_FLUID; v<NCOMP_TOTAL; v++) fcCon[f][v] = FMAX( fcCon[f][v], TINY_NUMBER );
 #     endif
    } // for (int f=0; f<6; f++)
 #  endif // #ifdef MHM_CHECK_PREDICT
+#endif // #ifndef SRHD
```

Comment:
The order of the comments is wrong. `\\ #ifdef MHM_CHECK_PREDICT` is before `// #ifndef SRHD`. Also, please add spaces to align the current indent.